deploy/osd-signature-updater/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Placeholder until there's a permanent cluster to run this.
```

Comment:
I thought the cluster to run this was going to be the appsre clusters. Is there a reason this is going into MCC now?

Reply:
This is a more general-purpose test, not for one particular cluster, but to ensure every cluster's scanning implementation is indeed scanning new pods and reporting its findings. I'll answer the slack specific questions on slack.